maths/krishnamurthy_number_2.py

```diff
@@ -0,0 +1,81 @@
+import doctest
+
+"""
+A Krishnamurthy number: A number that's equal to the sum of the factorials of its digits
+
+For instance, 145 = 1! + 4! + 5! ; 40585 = 4! + 0! + 5! + 8! + 5!, but 78 != 7! + 8!
+"""
```

Comment:
Plz add a Wikipedia or other informative link so that viewers can go directly for detailed understanding of this Krishnamurthy number.